eos-s3/requirements.txt

```diff
@@ -1,3 +1,3 @@
 python-constraint
 serial
-git+https://github.com/QuickLogic-Corp/quicklogic-fasm@57b6e60574a9d483dc94710d0d3ff42a62b4ec41
+git+https://github.com/QuickLogic-Corp/quicklogic-fasm@aaf4c314a165b6185b0983019d8aae4d0d4db6cb
```

Comment:
In order to have `f4pga` installed, you will need to add it here. See https://github.com/chipsalliance/f4pga-examples/blob/main/xc7/requirements.txt#L17. However, instead of `main`, you need to temporarily set it to the feature branch you are using in f4pga. E.g.:

`https://github.com/antmicro/f4pga/archive/pcza/ql-support.zip#subdirectory=f4pga`